admin/src/scenes/volontaires/edit.js

```diff
@@ -426,6 +429,25 @@ export default (props) => {
           </>
         )}
       </Formik>
+      {user.role === "admin" && (
+        <DeleteBtn
+          onClick={async () => {
+            if (!confirm("Êtes-vous sûr(e) de vouloir supprimer ce profil")) return;
+          try {
+            const { ok, code } = await api.remove(`/young/${young._id}`);
+            if (!ok && code === "OPERATION_UNAUTHORIZED") return toastr.error("Vous n'avez pas les droits pour effectuer cette action");
+            if (!ok) return toastr.error("Une erreur s'est produite :", code);
+            toastr.success("Ce profil a été supprimé.");
+            return history.push(`/inscription`);
+          } catch (e) {
+            console.log(e);
+            return toastr.error("Oups, une erreur est survenue pendant la supression du profil :", e.code);
+          }
+          }}
+        >
+          Supprimer
+        </DeleteBtn>
+      )}
```

Comment:
plus besoin de ca ducoup si ?

```suggestion
        <DeleteBtn
          onClick={async () => {
            if (!confirm("Êtes-vous sûr(e) de vouloir supprimer ce profil")) return;
          try {
            const { ok, code } = await api.remove(`/young/${young._id}`);
            if (!ok && code === "OPERATION_UNAUTHORIZED") return toastr.error("Vous n'avez pas les droits pour effectuer cette action");
            if (!ok) return toastr.error("Une erreur s'est produite :", code);
            toastr.success("Ce profil a été supprimé.");
            return history.push(`/inscription`);
          } catch (e) {
            console.log(e);
            return toastr.error("Oups, une erreur est survenue pendant la supression du profil :", e.code);
          }
          }}
        >
          Supprimer
        </DeleteBtn>
```